Philips.CodeAnalysis.Common/DiagnosticIds.cs

```diff
@@ -116,5 +116,6 @@ public enum DiagnosticIds
 		ThrowInformationalExceptions = 2121,
 		AvoidExceptionsFromUnexpectedLocations = 2122,
 		PassSenderToEventHandler = 2123,
+		DocumentUnhandledExceptions = 2124,

```

Comment:
PH2020: read "... from an NotImplementedException". Try different language to avoid "a" vs "an".

Reply:
Rephrased to avoid the "a" vs "an" issue